src/files.ts

```diff
@@ -249,49 +251,47 @@ export async function fetchProject(scriptId: string, rootDir = '', versionNumber
 /**
  * Pushes project files to script.google.com.
  */
-export async function pushFiles() {
+export async function pushFiles(force:boolean) {
   const { scriptId, rootDir } = await getProjectSettings();
   if (!scriptId) return;
-  getProjectFiles(rootDir, (err, projectFiles, files = []) => {
+  getProjectFiles(rootDir, async (err, projectFiles, files = []) => {
     if (err) {
-      logError(err, LOG.PUSH_FAILURE);
-      spinner.stop(true);
-    } else if (projectFiles) {
-      const [nonIgnoredFilePaths] = projectFiles;
-      const filesForAPI: any = files;
-      script.projects.updateContent(
-        {
+      return logError(err, LOG.PUSH_FAILURE);
+    }
+    const [nonIgnoredFilePaths] = projectFiles || [[]];
+    const filesForAPI: any = files;
+    script.projects.updateContent(
```

Comment:
Note: It would be nice to convert this to `await script.projects.updateContent`, but it's not required for this PR.